Extension/src/LanguageServer/client.ts

```diff
@@ -539,6 +540,8 @@ export interface TextDocumentWillSaveParams {
 interface InitializationOptions {
     packageVersion: string;
     extensionPath: string;
+    baseStoragePath: string | undefined;
```

Comment:
Can this `baseStoragePath` be only string too?

Reply:
Yes